src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -118,7 +116,7 @@ public void start(final Map<String, String> parsedConfig) {
   @Override
   public void stop() {
     setupComplete = false;
-    LOGGER.info(Logging.logMessage("SnowflakeSinkConnector:stop"));
```

Comment:
This probably could have been in a separate PR, but I refactored the logMessage function. It is now in Utils.java since we use it in places for not just logging. The LoggingHandler calls the new logMessage function, so these are removed

Reply:
just want to make sure there is no behavior changes with this refactor?

Reply:
None of this logging change should affect the connector functionality. LoggingHandler is just an additional layer that keeps track of the correlationId and injects it into the log string